hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -459,6 +495,9 @@ export class Om extends React.Component<Record<string, object>, IOmdbInsightsSta
     } else if (this.state.activeTab  === '4') {
       this.fetchDeletedKeys(this.state.DEFAULT_LIMIT, this.state.prevKeyDeleted);
     }
+      else if (this.state.activeTab === '5') {
+      this.fetchDeletePendingDir(this.state.DEFAULT_LIMIT, this.state.prevKeyDirDelete);
```

Comment:
@smitajoshi12 thanks for working on this patch. Pls correct the formatting and indent.

Reply:
Completed Changes